.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Please ensure your pull request follows these guidelines:
+
+- [ ] Link to the related issue with requirements and/or clearly describe 1) what problem this solves and 2) the requirements to review it against.
+- [ ] Update the `CHANGELOG.md` file with the added feature or fix. If there is not yet a heading for an upcoming release, add one following the established pattern.
+- [ ] Keep the pull request minimal! Break large updates into separate pull requests whenever possible. Small requests get reviewed more quickly.
+
```

Comment:
- [ ] Follow our style guide. Most modules have an eslint configuration, so you can run `npx eslint .` in the module if your editor does not automatically pick up on that.